src/js/pages/Setup.js

```diff
@@ -194,6 +197,15 @@ class Setup extends mixin(StoreMixin) {
     PreFlightStore.beginStage();
   }
 
+  getCheckboxLabel(label, tooltipContent) {
+    return (
+      <span>
+        {label}
+        <Tooltip content={tooltipContent} width={300} wrapText={true} />
```

Comment:
I think it would be better to update all tooltips at once because the component works differently (the change from `position: absolute` to `position: fixed`, scroll listeners, etc)